pkgs/by-name/da/dawarich/package.nix

```
@@ -0,0 +1,165 @@
+{
+  lib,
+  applyPatches,
+  bundlerEnv,
+  fetchFromGitHub,
+  fetchNpmDeps,
+  h3_3,
+  nixosTests,
+  nodejs,
+  npmHooks,
+  ruby_3_4,
+  stdenv,
+  tailwindcss_3,
+
+  sources ? ./. + "/sources.json",
+  gemset ? import (./. + "/gemset.nix"),
```

```suggestion
  sources ? ./sources.json,
  gemset ? import ./gemset.nix,
```

Remove the extra line and no need for the string manipulation.